tools/msvc/sbclang.targets

```diff
@@ -711,13 +727,6 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="ComputeRuntimeInputs" DependsOnTargets="ComputeClangOptions;ComputeStarboardRequiresObjectiveCRuntime">
-    <ItemGroup>
-      <Link Condition="'$(StarboardLinkObjCRuntime)' == 'true' and '$(StarboardRequiresObjectiveCRuntime)' == 'true'" Include="$(StarboardObjCRuntimeLib)"/>
```

Comment:
Again we should split it up correctly and possibly have libobjc2 build as part of the normal build instead of as a checked in binary. 

Nothing in this file ensures that its linked. I am making as much of this as simple as possible / not special. It is added to the linker in WinObjC.Language.targets like in all the other packages.